.github/actions/handle_docker/get_images_to_build.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
do we need a test to check for copyright header?

Reply:
Yes, we do. :-) Discussing this with @jacekpawlak right now.